orchestrator/nodes/notification/notification.html

```diff
@@ -0,0 +1,44 @@
+<script type="text/x-red" data-template-name="notification">
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="label.name"></label>
+        <input type="text" id="node-input-name">
+    </div>
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-edit"></i> <span data-i18n="label.message"></label>
```

Comment:
I think an option to this configuration be read from node configuration or a message's attribute will be useful.